src/ResourceManagement/Compute/Compute.Tests/Helpers/Helpers.cs

```diff
@@ -54,7 +54,7 @@ public static void DeleteIfExists(this IResourceGroupsOperations rgOps, string r
             }
         }
 
-        public static void ValidateVirtualMachineSizeListResponse(IPage<VirtualMachineSize> vmSizeListResponse)
+        public static void ValidateVirtualMachineSizeListResponse(IEnumerable<VirtualMachineSize> vmSizeListResponse)
```

Comment:
This return type has changed quite some times. Is this time the final change? @stankovski